main.yml

```diff
@@ -644,6 +646,20 @@ openshift:
 
   top_level: true
 
+ocp-advisor: 
+  title: Advisor
+  source_repo: https://github.com/RedHatInsights/ocp-advisor-frontend
+  deployment_repo: https://github.com/RedHatInsights/ocp-advisor-frontend-build
+  frontend:
+    module: ocp-advisor#./RootApp
+    isHidden: true
+    paths:
+      - /openshift/advisor
```

Comment:
There was a requirement for the new Advisor to run under the `/openshift/insights/advisor` URL, so it seems we need to rewrite it to the suggested format :)